src/panels/Scene/SceneGraphNode/SceneGraphNodeView.tsx

```diff
@@ -17,15 +15,20 @@ import styles from './SceneGraphNodeView.module.css';
 
 interface Props {
   uri: Uri;
-  closeable?: boolean;
+  // Extra content to be shown at the top right of the view
+  extraTopControls?: React.ReactNode;
```

Comment:
I can't really see a use case for this prop to be used for anything else than the close button 🤔 Especially where it is placed, so this seems overly general to me. 

I can see that it is conceptually nice to have the closing code in the scene panel, but then I would prefer to call this prop `closeButton` or something similar unless we can come up with another use case for it

Reply:
Myeah my problem was that to me it didnät feel like the "close button" belonged in the scene graph node view component.  In the suggestion I sent an image to you (where the close button was above the SGN header), I had moved it out of this view completely, but realized that this led to way to much screen space being wasted

Ideally, I would not want to have this in here at all

Reply:
I agree to both accounts :) maybe we can split the scene graph node view component into two parts. one with the tabs and one with the header. That way, we can make a tiny component for the SceneGraphNodeView that is used in windows standalone, and just use the tab component in the scene menu

Reply:
Yeah I think that would be better :)  I've been thinking that we maybe want to use a different header component for the SGN View anyway. Using the same one as in the list feels pretty limiting as of now